app/views/layouts/base.html.erb

```diff
@@ -16,8 +16,10 @@
     <%= yield(:stylesheets) %>
 
     <%= csrf_meta_tags %>
+    <%= javascript_include_tag *webpack_asset_paths("bundle") %>
```

Comment:
This is the culprit for what I mentioned in the other comment, it returns `http://localhost:3808/webpack/bundle.js`, the problem comes from here, https://github.com/mipearson/webpack-rails/blob/27f661f671a514c9a681de07d70854c0473113bc/lib/webpack/rails/helper.rb#L24 , you need to set `devServer.host' to fix it